profile_bluesky/startup/instrument/devices/high_field_magnet.py

```diff
@@ -4,137 +4,93 @@
 
 __all__ = ['mag6t']
 
-from ophyd import Component, Device, EpicsMotor, PVPositioner
-from ophyd import EpicsSignal, EpicsSignalRO, Signal
-from ophyd import Kind
-from ophyd.status import wait as status_wait
+from ophyd import (Component, Device, EpicsMotor, PVPositioner, EpicsSignal,
+                   EpicsSignalRO, Signal)
 from ..framework import sd
 from ..session_logs import logger
 logger.info(__file__)
 
 
 class AMIZones(Device):
     high_field = Component(EpicsSignal, "Field.VAL", write_pv="FieldSet",
-                           kind=Kind.config)
+                           kind="config")
     ramp_rate = Component(EpicsSignal, "Rate", write_pv="RateSet",
-                          kind=Kind.config)
+                          kind="config")
 
 
 class AMIController(PVPositioner):
     """ Magnetic field controler """
 
     # position
     readback = Component(EpicsSignalRO, "Field", auto_monitor=True,
```

Comment:
Don't know if you saw it with the EpicsMotor but I like the way that is configured.  In the `__init__()` method, after it calls `super().__init__(...)`,

```python
self.readback.name = self.name
```

so this makes the `.read()` method report `motor=..., motor.setpoint=...` where the first value is actually `motor.readback`.  The column labels are simpler then.  You might consider that here.

Reply:
This is a good one. Should this be also the standard for `ophyd.PVPositioner`?

Reply:
My opinion is yes.  But it is a preference.